excel_to_json_partners.py

```
@@ -0,0 +1,153 @@
+                    continue  # Skip row if name is missing or blank
+
+                raw_src = row[headers.index(expected_columns[1])] or ''
+                logo_url = convert_drive_link_to_direct_url(raw_src)
+
+                name_clean = str(raw_name).lower().replace(" ", "")
+                row_data = {
+                    'id': name_clean,
+                    'src': logo_url,
+                    'alt': name_clean,
+                    'name': str(raw_name).strip(),
+                    'src': logo_url,
```

Ruff (0.12.2)

Dictionary key literal `'src'` repeated

Remove repeated key literal `'src'`

(F601)